entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ -n "${DBT_PROFILES}" ]; then
+  echo "${DBT_PROFILES}" > /.dbt/profiles.yml
+elif [ -f ~/.dbt/profiles.yml ]; then
+  cp ~/.dbt/profiles.yml /.dbt/profiles.yml
+fi
+
+if [ -n "${AWS_CREDENTIALS}" ]; then
+  echo "${AWS_CREDENTIALS}" > /.dbt/aws_credentials
+elif [ -f ~/.aws/credentials ]; then
+  cp ~/.aws/credentials /.dbt/aws_credentials
+fi
+
+if [ -n "${BQ_KEYFILE}" ]; then
+  echo "${BQ_KEYFILE}" > /.dbt/bq_keyfile.json
+fi
```

Comment:
No fallback for BQ as the reference to it is in `profiles.yml` and so up to the user to make sure the path is correct to a file that exists within the container, be it via mounting or this env var.

Reply:
This is probably fine. The ENVs should be passed together (either both or neither). If the ENVs are passed, we can assume they agree about the location. Otherwise, we can assume the mounted files agree.